clang/lib/Basic/Targets/X86.cpp

```diff
@@ -954,6 +954,8 @@ void X86TargetInfo::getTargetDefines(const LangOptions &Opts,
     Builder.defineMacro("__CCMP__");
   if (HasCF)
     Builder.defineMacro("__CF__");
+  if (HasEGPR && HasPush2Pop2 && HasPPX && HasNDD)
```

Comment:
Add comment
`Condition here is aligned with the feature set of mapxf in Options.td`

Reply:
4929d88